metrics-proxy/src/main/java/ai/vespa/metricsproxy/node/CoredumpMetricGatherer.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package ai.vespa.metricsproxy.node;
+
+import ai.vespa.metricsproxy.metric.model.ConsumerId;
+import ai.vespa.metricsproxy.metric.model.MetricsPacket;
+import ai.vespa.metricsproxy.metric.model.ServiceId;
+import com.yahoo.vespa.defaults.Defaults;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.util.Set;
+import java.util.logging.Logger;
+
+
+/**
+ * @author olaa
+ */
+public class CoredumpMetricGatherer {
+
+    private static final Path COREDUMP_PATH = Path.of(Defaults.getDefaults().underVespaHome("var/crash/processing"));
+
+    private static final Logger logger = Logger.getLogger(CoredumpMetricGatherer.class.getSimpleName());
+
+
+    protected static MetricsPacket.Builder gatherCoredumpMetrics(FileWrapper fileWrapper) {
+        int coredumps = getCoredumpsFromLastPeriod(fileWrapper);
```

Comment:
Ahh yes, the original check only considers files under a certain age, but since this age was quite high, I decided to remove it in the last commit. Will rename.